Dockerfile

```diff
@@ -12,7 +12,7 @@
 #
 
 # ----------- deps -----------
-FROM python:slim-stretch as deps
+FROM python:slim as deps
```

Comment:
should we not pin to specific python version?

Reply:
Noted, so we don't face the same issue as yesterday.
Thanks for the catch @JP-Globality